lib/internal/Magento/Framework/Css/PreProcessor/Instruction/MagentoImport.php

```diff
@@ -57,25 +58,33 @@ class MagentoImport implements PreProcessorInterface
      */
     private $themeProvider;
 
+    /**
+     * @var ModuleManagerInterface
+     */
+    private $moduleManager;
+
     /**
      * @param DesignInterface $design
      * @param CollectorInterface $fileSource
      * @param ErrorHandlerInterface $errorHandler
      * @param \Magento\Framework\View\Asset\Repository $assetRepo
      * @param \Magento\Framework\View\Design\Theme\ListInterface $themeList
+     * @param ModuleManagerInterface $moduleManager
      */
     public function __construct(
         DesignInterface $design,
         CollectorInterface $fileSource,
         ErrorHandlerInterface $errorHandler,
         \Magento\Framework\View\Asset\Repository $assetRepo,
-        \Magento\Framework\View\Design\Theme\ListInterface $themeList
+        \Magento\Framework\View\Design\Theme\ListInterface $themeList,
+        ModuleManagerInterface $moduleManager
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
done.